easybuild/easyconfigs/g/GROMACS/GROMACS-2023.2-cpeGNU-22.12-HeFFTe-GPU.eb

```diff
@@ -0,0 +1,127 @@
+# Based on recipe from Rasmus Kronberg (CSC) 
+#
+easyblock = 'CMakeMake'
+
+name =          'GROMACS'
+version =       '2023.2'
+versionsuffix = '-HeFFTe-GPU' 
+
+homepage = 'http://www.gromacs.org'
+
+whatis = [
+    'Description: GROMACS (with multi-GPU acceleration), a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.',
+]
+
+description = """
+GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the
+Newtonian equations of motion for systems with hundreds to millions of
+particles. This version enables GPU acceleration with PME decomposition feature 
+allowing distribution across multiple nodes and GPUs (supports ROCm via hipSYCL).
+
+Binaries included:
+* gmx_mpi: Multi-node MPI version with HeFFTe for a multi-GPU decomposition.
+
+The module sets the environment variables that would otherwise be set by running
+the script $EBROOTGROMACS/bin/GMXRC. So running this script snould not be
+}necessary in most cases.
+
+The module also defines the shell function gromacs-completion that can be used to
+turn bash command completion on for GROMACS. Note that unloading the module will
+not turn this off, so it may have consequences for things you do later on or
+conflict with the command completion of a different version of GROMACS.
+
+"""
+
+toolchain = {'name': 'cpeGNU', 'version': '22.12'}
+toolchainopts = {'opt': True, 'usempi': True, 'verbose': False, 'pic': True, 'openmp': True}
+
+source_urls = ['https://ftp.gromacs.org/pub/gromacs/']
+sources =     [SOURCELOWER_TAR_GZ]
+
+patches = [
+    'gromacs_cmake_manage_sycl.patch',
+]
+
+builddependencies = [
+    ('buildtools', '%(toolchain_version)s', '', True), 
+]
+
+dependencies = [
+    ('hipSYCL', '0.9.4', '-rocm-5.2.3'), 
+    ('HeFFTe', '2.3.0', '-rocm-5.2.3'),
```

Comment:
In the meantime 2.4.0 has been released. I've not tested, but it claims perf improvements so it could be useful to test (in follow-up).